tests/Doctrine/Tests/Models/PersistentObject/PersistentEntity.php

```diff
@@ -29,6 +30,8 @@ class PersistentEntity extends PersistentObject
     protected $name;
 
     /**
+     * @ORM\JoinColumn(nullable=true)
```

Comment:
Why is this annotation necessary now?

Reply:
Not sure, I copied that from the #10547 PR. Let's see if tests pass without it.